content/v1.15/software/install.md

```diff
@@ -264,18 +247,29 @@ at the table below.
 {{< table caption="Feature flags" >}}
 | Status | Flag | Description |
 | --- | --- | --- |
-| Beta | `--enable-composition-revisions` |Enable support for CompositionRevisions |
-| Alpha | `--enable-composition-functions` | Enable support for Composition Functions. |
-| Alpha | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas. |
+| Beta | `--enable-composition-functions` | Enable support for Composition Functions. |
+| Beta | `--enable-composition-revisions` | Enable support for CompositionRevisions. |
+| Beta | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas. |
 | Alpha | `--enable-environment-configs` | Enable support for EnvironmentConfigs. |
 | Alpha | `--enable-external-secret-stores` | Enable support for External Secret Stores. |
+| Alpha | `--enable-usages` | Enable support for Usages. |
+| Alpha | `--enable-realtime-compositions` | Enable support for real time compositions. |
 {{< /table >}}
 {{< /expand >}}
 
 Set these flags either in the `values.yaml` file or at install time using the
 `--set` flag, for example: `--set
 args='{"--enable-composition-functions","--enable-composition-webhook-schema-validation"}'`.
 
+#### Change the default package registry
+
+Beginning with Crossplane version 1.15.0 Crossplane downloads packages from the
+[Upbound Marketplace](https://marketplace.upbound.io) at `xpkg.upbound.io` 
+instead of DockerHub. 
+
+Change the default registry location during the Crossplane install with 
```

Comment:
Did you also want to include content around how to clean-up/fix the scenario where folks miss the notification about the default registry changing and they upgrade to v1.15 without taking that into account?

e.g. the very raw content from https://gist.github.com/jbw976/793210f26f2dfbb458a41d5dc437ce7e?

Reply:
I want to bring this into a new "upgrade" chapter that I'm going to write shortly after release, since the context for this is only related to upgrades and not new installs.

Reply:
Sounds good - a fast follow on that would be very appreciated, because I know over the next few days we'll have at least some folks that need to get sorted out with their packages and registry default 😞 - so being able to efficiently support them (e.g. a consistent docs spot to point them at) will be really helpful! 🙇‍♂️